recipes/winflexbison/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "winflexbison" %}
+{% set version = "2.5.20" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/lexxmark/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: cd29f888f167128c22211444c6e31b5f4823a7de0e23d8039559ee87b80349c9
+    patches:
+      - fix-install.patch
+
+build:
+  number: 0
+  skip: true  # [not win or py2]
```

Comment:
```suggestion
  skip: true  # [not win or vc<14]
```

Reply:
Thanks for the tips. The `vc` selector can be used in this case.